.github/CODEOWNERS

```diff
@@ -1,247 +1,309 @@
-.editorconfig @BenHenning
-/.idea/ @BenHenning
-
-# Robolectric configuration.
-*.properties @BenHenning
-
-# Firebase configuration.
-/app/google-services.json @BenHenning
 
-# Binary files.
-*.png @BenHenning
 
-# Configurations for Bazel-built Android App Bundles.
-bundle_config.pb.json @BenHenning
-
-# Important codebase files.
-LICENSE @BenHenning
-NOTICE @BenHenning
-
-# Language configuration files.
-config/**/alllanguages/*.textproto @BenHenning
-config/**/productionlanguages/*.textproto @BenHenning
-
-# Configuration for KitKat-specific curated builds.
-config/kitkat_main_dex_class_list.txt @BenHenning
-
-#####################################################################################
-#                                    app module                                     #
-#####################################################################################
-
-# Global app module code ownership.
-/app/**/*.kt @BenHenning
-/app/**/*.java @BenHenning
-
-# State players.
-/app/src/*/java/org/oppia/android/app/player/ @BenHenning
-
-# Bindable adapter utilities.
-/app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt @BenHenning
-/app/src/main/java/org/oppia/android/app/recyclerview/RecyclerViewBindingAdapter.java @BenHenning
-/app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt @BenHenning
-
-# Dagger configuration.
-/app/src/main/java/org/oppia/android/app/activity/ @BenHenning
-/app/src/main/java/org/oppia/android/app/application/ @BenHenning
-/app/src/main/java/org/oppia/android/app/fragment/ @BenHenning
-/app/src/main/java/org/oppia/android/app/view/ @BenHenning
+# ---=============================== MODULES ==============================--- #
 
-# Databinding adapters.
-/app/src/main/java/org/oppia/android/app/databinding/ @BenHenning
 
-# App notices functionality (such as for deprecations).
-/app/src/*/java/org/oppia/android/app/notice/ @BenHenning
 
-# Parsing functionality needed for interactions.
-/app/src/*/java/org/oppia/android/app/parser/ @BenHenning
+################################################################################
+#                                  app module                                  #
+################################################################################
 
-# Bazel/data-binding shims.
-/app/src/*/java/org/oppia/android/app/shim/ @BenHenning
+# All app code files.
+/app/**/*.kt @oppia/android-frontend-reviewers
+/app/**/*.java @oppia/android-frontend-reviewers
 
-# Splash screen.
-/app/src/*/java/org/oppia/android/app/splash/ @BenHenning
+# All app resource files.
+/app/src/main/res/**/*.xml @oppia/android-frontend-reviewers
 
-# View model infrastructure.
-/app/src/*/java/org/oppia/android/app/viewmodel/ @BenHenning
-
-# App testing infrastructure.
-/app/src/*/java/org/oppia/android/app/testing/ @BenHenning
+# Exemptions to resource shrinking.
+/app/src/main/res/raw/shrink_exemptions.xml @oppia/android-developer-infrastructure-reviewers
```

Comment:
Can we call this android-dev-workflow-reviewers instead, to (a) increase distinguishability from the "app infrastructure" team, and (b) mimic Web's "dev workflow" team naming? (It's also shorter, which would make @-mentioning the team a little more readable, but that's a minor benefit.)

Reply:
Seems reasonable, done.

Reply:
No follow-up comment and you mentioned LGTM @seanlip, so resolving this thread.